docker-compose.yml

```diff
@@ -67,7 +67,7 @@ services:
   orbs-e2e:
     image: orbs:e2e
     environment:
-      API_ENDPOINT: http://orbs-network-node-3:8080/api/
+      API_ENDPOINT: http://orbs-network-node-1:8080/api/
```

Comment:
We need to revert this change so we could test that non-leader node gets updated.

Reply:
yes, I remember